.github/workflows/bsim-tests.yaml

```diff
@@ -36,8 +39,41 @@ jobs:
       BSIM_OUT_PATH: /opt/bsim/
       BSIM_COMPONENTS_PATH: /opt/bsim/components
       EDTT_PATH: ../tools/edtt
-      bsim_test_results_file: ./bsim_out/bsim_results.xml
+      bsim_bluetooth_test_results_file: ./bsim_bluetooth/bsim_results.xml
+      bsim_networking_test_results_file: ./bsim_net/bsim_results.xml
     steps:
+      - name: Check if Bluethooth files changed
+        uses: tj-actions/changed-files@v35
+        id: check-buetooth-files
+        with:
+          files: |
+            .github/workflows/bsim-tests.yaml
+            .github/workflows/bsim-tests-publish.yaml
+            west.yml
+            subsys/bluetooth/**
+            tests/bsim/bluetooth/**
```

Comment:
Please trigger also on changes in tests/bsim/* (otherwise we don't check changes in the common scripts) in at least one of the 2 groups